README.md

```diff
@@ -66,8 +77,7 @@ If you want to help with the development (improvement, update, bug-fixing, ...)
 
 We appreciate any contribution and if a release is needed to be done on PyPI, please just contact one of us.
 
-Development
------------
+### Development
 
 You need a working `python` environment (system, virtualenv, pyenv, etc) version 3.7 or superior.
```

Comment:
Python is a proper noun, not an inline literal.

Also check the Python version.

```suggestion
You need a working Python environment version 3.7 or later.
```